File: Archive/fizzbuzz/perl/fizzbuzz.pl
use strict;
use warnings;
use feature qw(say);

for my $i (1..15000000) {
    say $i % 15 == 0 ? "FizzBuzz"
      : $i %  3 == 0 ? "Fizz"
      : $i %  5 == 0 ? "Buzz"
      : $i;
}

File: Archive/fizzbuzz/java/fizzbuzz.java
public class FizzBuzz {
    public static void main(String[] args) {
        for (int number = 1; number <= 2000000; number++) {
            if (number % 15 == 0) {
                System.out.println("FizzBuzz");
            } else if (number % 3 == 0) {
                System.out.println("Fizz");
            } else if (number % 5 == 0) {
                System.out.println("Buzz");
            } else {
                System.out.println(number);
            }
        }
    }
}
